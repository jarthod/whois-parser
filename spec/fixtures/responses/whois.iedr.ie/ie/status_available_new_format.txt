Not found: example-nonexistent-domain-123456.ie

% Important Notice
% If you believe that information published on this page is incorrect, or should not be published,please contact your Registrar, or the .IE Registration Services Department who will advise you further.
% You can also contact your Registrar or .IE if your Registrant name is not showing and you would like it to be published.
% Our contact information is available at https://www.weare.ie/contact-us/
% Rights restricted by copyright: https://www.weare.ie/registrant-terms-conditions-policy/
% Do not remove this notice
