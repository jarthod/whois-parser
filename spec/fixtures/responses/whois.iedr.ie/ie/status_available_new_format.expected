#disclaimer
  %s == "Important Notice If you believe that information published on this page is incorrect, or should not be published,please contact your Registrar, or the .IE Registration Services Department who will advise you further. You can also contact your Registrar or .IE if your Registrant name is not showing and you would like it to be published. Our contact information is available at https://www.weare.ie/contact-us/ Rights restricted by copyright: https://www.weare.ie/registrant-terms-conditions-policy/ Do not remove this notice"

#domain
  %s == "example-nonexistent-domain-123456.ie"

#domain_id
  %s == nil

#status
  %s == :available

#available?
  %s == true

#registered?
  %s == false

#created_on
  %s == nil

#updated_on
  %s == nil

#expires_on
  %s == nil

#registrar
  %s == nil
